lib/common/xml.c

```diff
@@ -2974,6 +2974,32 @@ pcmk__xe_set_props(xmlNodePtr node, ...)
     va_end(pairs);
 }
 
+void
+pcmk__xe_set_bool_attr(xmlNodePtr node, const char *name, bool value)
+{
+    crm_xml_add(node, name, value ? XML_BOOLEAN_TRUE : XML_BOOLEAN_FALSE);
+}
+
+bool
+pcmk__xe_get_bool_attr(xmlNodePtr node, const char *name)
+{
+    return crm_is_true(crm_element_value(node, name));
+}
+
+bool
+pcmk__xe_bool_attr_is_false(xmlNodePtr node, const char *name)
+{
+    const char *value = crm_element_value(node, name);
+
+    if (value != NULL) {
+        int ret;
+        int rc = crm_str_to_boolean(value, &ret);
+        return rc == 1 && ret == FALSE;
+    } else {
+        return false;
+    }
+}
+
```

Comment:
These should go in nvpair.c

Reply:
Would this then require a new include/crm/common/nvpair_internal.h file?

Reply:
Existing ones are in common/internal.h but nothing wrong with separating them if you want

Reply:
Meh, might as well keep them all in common/internal.h so you have something to do when it comes time for another best practices PR.